src/GovUk.Education.ExploreEducationStatistics.Notifier/PublicationNotifier.cs

```diff
@@ -82,9 +90,14 @@ public class PublicationNotifier
                     var values = new Dictionary<string, dynamic>
                     {
                         // Use values from the queue just in case the name or slug of the publication changes
-                        {"publication_name", publicationNotification.Name},
-                        {"publication_link", webApplicationBaseUrl + "find-statistics/" + publicationNotification.Slug},
-                        {"unsubscribe_link", baseUrl + entity.PartitionKey + "/unsubscribe/" + unsubscribeToken}
+                        { "publication_name", notificationMessage.PublicationName },
+                        { "release_name", notificationMessage.ReleaseName },
+                        {
+                            "release_link",
+                            $"{webApplicationBaseUrl}find-statistics/{notificationMessage.PublicationSlug}/{notificationMessage.ReleaseSlug}"
+                        },
+                        { "update_note", notificationMessage.UpdateNote },
+                        { "unsubscribe_link", baseUrl + entity.PartitionKey + "/unsubscribe/" + unsubscribeToken }
```

Comment:
Could use string interpolation here to be consistent with how we create the `release_link` variable.